junit5/src/main/java/org/jboss/weld/junit5/auto/TestInstanceInjectionTarget.java

```diff
@@ -0,0 +1,28 @@
+package org.jboss.weld.junit5.auto;
+
+import javax.enterprise.context.spi.CreationalContext;
+import javax.enterprise.inject.spi.InjectionTarget;
+
+import org.jboss.weld.injection.ForwardingInjectionTarget;
+
+class TestInstanceInjectionTarget<T> extends ForwardingInjectionTarget<T> {
```

Comment:
In fact, on closer look, all changes made to `TestInstanceInjectionExtension` are basically just a swap from singular test instance injection to potential list of classes to inject into.

Reply:
I'm afraid it could not be an anonymous inner class as well which I initially also thought due to

> [ERROR] The class org.jboss.weld.junit5.auto.TestInstanceInjectionExtension$2 could be refactored into a named _static_ inner class [org.jboss.weld.junit5.auto.TestInstanceInjectionExtension] At TestInstanceInjectionExtension.java:[line 99] SIC_INNER_SHOULD_BE_STATIC_ANON

https://travis-ci.org/github/weld/weld-junit/jobs/702941251

Apart from that I hope I haven't changed anything significant really.

Reply:
> I'm afraid it could not be an anonymous inner class

I meant named inner class, that shouldn't give you any troubles. E.g.
```
public class TestInstanceInjectionExtension implements Extension {
  // extension code
  private class class TestInstanceInjectionTarget<T> extends ForwardingInjectionTarget<T> {
    ...
  }
}
```